plugins/source/gandi/docs/tables/gandi_certificate_packages.md

```diff
@@ -0,0 +1,22 @@
+# Table: gandi_certificate_packages
+
+
+
+The primary key for this table is **name**.
+
+
+## Columns
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|sharing_id|String|
```

Comment:
Should `sharing_id` maybe be part of the PK here?

Reply:
This is a "global" table with values like:
```
sharing_id      |
name            | cert_std_3_0_0
name_label      |
href            | https://api.gandi.net/v5/certificate/packages/cert_std_3_0_0
max_domains     | 3
type            |
type_label      |
wildcard        | f
```
it would be the same rows repeated if `sharing_id` was a key. The `sharing_id` column is provided because if the user sets it in the config it's passed down to the client automatically so filtering is probably done on the server side.